client/src/components/OverviewPage/EventModeButton/index.tsx

```diff
@@ -50,11 +50,11 @@ export const EventModeButton = observer(() => {
 
         {seatCount !== 0 && (
           <input
-            className="basis-1/2 px-2 py-1 mx-1 mt-1 bg-green-150 text-white text-sm font-medium rounded-md shadow-sm hover:bg-green-700 focus:outline-none focus:ring-2 focus:ring-green-500 focus:ring-offset-2"
+            className="basis-1/2 px-2 py-1 mx-1 mt-1 bg-green-150 text-black text-sm font-medium rounded-md shadow-sm"
```

Comment:
https://tailwindcss.com/docs/background-color#changing-the-opacity here you can check which number you can use for  bg-green-### , not all the number are allow. I would suggest to change the number to another one which make sense. bg-green-150 doesnt exist and that is why it is white in the background. text-white is OK